src/Player.Vm.Api/Player.Vm.Api.csproj

```diff
@@ -7,7 +7,7 @@
   <ItemGroup>
     <PackageReference Include="AutoMapper" Version="9.0.0" />
     <PackageReference Include="Microsoft.OpenApi" Version="1.2.3" />
-    <PackageReference Include="Player.Api.Client" Version="1.7.0" />
+    <PackageReference Include="Player.Api.Client" Version="1.7.0-dev.6" />
```

Comment:
If this is reliant on the other PR to Player.Api, we should release a new Client version when we're ready and update here before merging

Reply:
good catch. agree.